<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_showlist</class>
 <widget class="QDialog" name="Dialog_showlist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>321</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>399</width>
      <height>319</height>
     </rect>
    </property>
    <widget class="QTextBrowser" name="showlist_textBrowser">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>70</y>
       <width>361</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="showlist_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="show_pushButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="show_lineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
